src/main/java/com/twilio/rest/api/v2010/account/conference/Participant.java

```diff
@@ -140,8 +140,8 @@ public static ParticipantUpdater updater(final String pathConferenceSid,
      */
     public static ParticipantCreator creator(final String pathAccountSid,
                                              final String pathConferenceSid,
-                                             final com.twilio.type.PhoneNumber from,
-                                             final com.twilio.type.PhoneNumber to) {
+                                             final com.twilio.type.Endpoint from,
```

Comment:
I'm not sure I understand the purpose/use case for this change. These types are autogenerated from our internal API definition, and the type expected here is a `com.twilio.type.PhoneNumber`.

Reply:
Ah I found the docs mentioning the alternate `to`/`from` formats [here](https://www.twilio.com/docs/voice/api/conference-participant-resource#create-a-participant)